templates/base.html

```diff
@@ -51,7 +51,9 @@
 {% block body %}{% endblock %}
 </main>
 
-
+<footer>
+  <p>© Cafébabel — 2017</p>
+</footer>
 
 <script src=https://rawgit.com/cafebabel/styleguide/master/script.js></script>
```

Comment:
This is also a script that should be ported into this repo

Reply:
sure